scripts/utils/trading_strategy_helpers.js

```diff
@@ -166,6 +167,25 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return tokenPromises
   }
 
+  /**
+   * Retrieves token information foll all tokens involved in the deposits.
+   * @param {Deposit[]} depositList List of {@link Deposit}
+   * @return {Promise<TokenObject>[]} list of detailed/relevant token information
+   */
+  const fetchTokenInfoForDeposits = function(deposits, debug = false) {
```

Comment:
Why don't we call this function fetchTokenInfo and then use it for deposits and withdrawals?